CONTRIBUTING.md

```diff
@@ -329,6 +329,11 @@ $ yarn watch & # runs in the background
   [conventionalcommits](https://www.conventionalcommits.org).
   * The title must begin with `feat(module): title`, `fix(module): title`, `refactor(module): title` or
     `chore(module): title`.
+    * `feat`: indicates a feature added (requires tests and README updates)
+    * `fix`: indicates a bug fixes (requires tests)
+    * `docs`: indicates updated documentation (docstrings or Markdown files)
+    * `refactor`: indicates a feature-preserving refactoring
+    * `chore`: something without directly visible user benefit (does not end up in the CHANGELOG). Typically used for build scripts, config, or changes so minor they don't warrant showing up the CHANGELOG.
```

Comment:
```suggestion
    * `chore`: something without directly visible user benefit (does not end up in the CHANGELOG). Typically used for build scripts or config changes.
```

I think "minor" changes leaves too much open to interpretation. I think if there is any functionality change, no matter how "minor", it should not be a chore.

Reply:
I'm not sure. New enum members for example, or a property that should always have been there but wasn't. I mean I guess technically they are features, but it feels a little like scraping the bottom of the barrel to me to put that up in the CHANGELOG. As in, I would feel ashamed putting it there 🫣

Reply:
Properties that should have been there like these? I feel like it's better to default to being less ambiguous. Core members can always change it to `chore` before approving if they really don't want it to show up in the CHANGELOG.

https://github.com/aws/aws-cdk/blob/6b4f92f2437c7ff782c88ce23925a04168728d7c/packages/%40aws-cdk/aws-ec2/lib/launch-template.ts#L621-L655

But if you think we should be using `chore` more for minor changes then I guess I'm fine with that.